### DISCUSSION TITLE
Feedback: data-importer documentation

### DISCUSSION THREAD

**Discussion by f-fritz** [firefly-iii/firefly-iii#6846] [General]
Hey

I just installed both firefly-iii and the data-importer via the self hosted / from sources guide in the documentation. I used @Engr-AllanG s [gist for the proxmox installation](https://gist.github.com/Engr-AllanG/34e77a08e1482284763fff429cdd92fa) (which is linked in the [official documentation](https://docs.firefly-iii.org/firefly-iii/installation/self_hosted/#ingredients)) as it includes an exemplary nginx configuration.

When installing the data-importer, I reused that nginx configuration and got an 502 Error right after accepting the oauth request when accessing the callback URL. The solution for the problem can be found in [#4101](https://github.com/firefly-iii/firefly-iii/issues/4101#issuecomment-735552487) and in the second gist for the data-importer.

Maybe I missed something, but I'd like suggest to enhance the data-importer documentation with the following information to save the lesser experienced user some headaches:
- Link @Engr-AllanG s [gist for the data-importer](https://gist.github.com/Engr-AllanG/e87f827092e3a6b876b912cd897428ae)
- Optionally add a little note to the [firefly-iii gist](https://gist.github.com/Engr-AllanG/34e77a08e1482284763fff429cdd92fa) saying that a modification for the nginx config is required when trying to use it for the data-importer
- Edit the FAQ / [Frequently seen issues page](https://docs.firefly-iii.org/data-importer/errors/freq_errors/#502-bad-gateway-errors) to not only say that the 502 error might be induced by a reverse proxy but to increase the buffer size of the actual server as well

Thanks!

**Comment by JC5** [comment]
Nice feedback, thanks! I'll pick it up 👍🏻

**Reply by JC5** [reply]
Done!
